UM/View/RenderBatch.py

```diff
@@ -288,10 +292,12 @@ def _renderItem(self, item):
 
         if mesh.hasIndices():
             if self._render_range is None:
+                element_count = mesh.getFaceCount();
                 if self._render_mode == self.RenderMode.Triangles:
-                    self._gl.glDrawElements(self._render_mode, mesh.getFaceCount() * 3 , self._gl.GL_UNSIGNED_INT, None)
-                else:
-                    self._gl.glDrawElements(self._render_mode, mesh.getFaceCount(), self._gl.GL_UNSIGNED_INT, None)
+                    element_count = mesh.getFaceCount() * 3
+                elif self._render_mode == self.RenderMode.TrianglesAdjacency:
+                    element_count = mesh.getFaceCount() * 3 * 4
```

Comment:
Shall we still have the `else` at the end? Then other cases won't be affected. Like this:
```
else:
    element_count = mesh.getFaceCount()
```